lib/has_my_gsuite_been_pwned/wrappers/have_i_been_pwned.ex

```diff
@@ -1,22 +1,20 @@
 defmodule HasMyGsuiteBeenPwned.HaveIBeenPwned do
+  @moduledoc """
+  Wrapper for the ex_pwned allowing us to retry requests if they timeout
+  """
+
   alias HasMyGsuiteBeenPwned.User
 
   def check_user(user = %User{}) do
-    user.email
-    |> ExPwned.Breaches.breachedaccount
-  end
-
-  def check_user_and_sleep(user = %User{}) do
-    case check_user(user) do
-      {:ok, breach_report, retry_after} ->
-        if (retry_after) do
-          # Sleep for retry_time + 150ms (to avoid getting hit by rate limit)
-          sleep_time = (retry_after[:retry_after] * 1000) + 150
-          :timer.sleep(sleep_time)
-        end
-
-        get_simplified_breach_report(breach_report, user)
-      _ -> nil
+    case ExPwned.Breaches.breachedaccount(user.email) do
+      {:ok, %{msg: "no breach was found for given input"}, _} ->
+        nil
+      {:ok, breach_report, _ } ->
+        breach_report
```

Comment:
Hey @zpeters, thanks for the awesome work again! Shouldn't we also wait for the `retry_after` in all cases? (Since the API responds with it in all cases)

Reply:
I agree with this.  I think it will be easy.  I'm going to try to push an update to this PR.  If i can't figure that out i'll send a new pr

Reply:
I think i'm missing something.  Can you show me other places you'd want to wait?

Reply:
So my assumption is that whenever a response comes back from haveibeenpwned.com, it also sends the retry after header.

For example this line `{:ok, breach_report, _ }`, the third variable is actually the `retry_after` (and it was being used as that before).

I think we should wait whenever the API responds back to us with data and we know that we're going to call it again.

In your code you're only waiting if we get a rate error back, but if we wait in the positive case as well, we should be able to avoid getting the rate error in the first place, right?

or am I completely mistaken (sorry) and the expwned library should be taking care of that now after your new changes to it?

Reply:
Definitely you are right.  I missed that.